### ISSUE TITLE
Reverse geocoding for google provider doesn't work

### ISSUE THREAD

**Issue by akittas** [DenisCarriere/geocoder#240]
When running the example from the docs:

```python
import geocoder
g = geocoder.google([45.15, -75.14], method='reverse')
```

reverse geocoding does not complete and the geocoding object returns: `<[INVALID_REQUEST] Google - Reverse>`.
It seems that the url is malformed. I will try to submit a PR within the weekend.

**Comment by DenisCarriere** [comment]
👍 Strange, this might be a recent Google change since `INVALID_REQUEST` is not coming from this python library.

**Comment by kasir003** [comment]
This change is not yet reflected in "pip".

**Comment by DenisCarriere** [comment]
Published new release `1.22.4`.

https://pypi.python.org/pypi/geocoder/1.22.4

**Comment by akittas** [comment]
Great, since this is fixed  now I'm closing the issue.
